Enterprise-CMCS/eAPD
# 4426 Adding match rate picker component

Resolves #4226 

### Description
Updates the fed-state split selector in Budget and FFP to use the new match rate selector.

### Significant changes or possible side effects
Updated the schema for costAllocation and keyStatePersonnel.split to have a value called fundingCategory. Updated existing APDs to have defaults for these values

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |

### Steps to manually verify this change

1. open a MMIS APD
2. navigate to the Budget and FFP page of an activity
3. select a split and funding rate (75/25 and 50/50 will not save until the funding category is selected)
4. verify that this value is saved

### This pull request is ready to code review when

      tests, refactor

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
